examples/ui/text.rs

```diff
@@ -74,7 +73,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
             }),
         ])
         .with_style(Style {
-            align_self: AlignSelf::FlexEnd,
+            align_self: AlignSelf::FlexStart,
```

Comment:
We can probably remove this, it will probably look the same. Maybe we can even remove the entire `.with_style` then.

Reply:
This one looks the same without the with_style(). Removed!